packages/ppsdk-integration/src/NoUI.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { useEffect } from 'react';
+
+import { PaymentMethodProps } from '@bigcommerce/checkout/payment-integration-api';
+
+import { usePropsToOnMount } from './usePropsToOnMount';
+
+export const NoUI = (props: PaymentMethodProps) => {
+    const onMount = usePropsToOnMount(props);
+
+    useEffect(onMount, [onMount]);
+
+    // eslint-disable-next-line react/jsx-no-useless-fragment
+    return <></>;
+};
```

Comment:
```suggestion
export const NoUI: React.FC<PaymentMethodProps> = (props) => {
  const onMount = usePropsToOnMount(props);
  
  useEffect(onMount, [onMount]);
  
  return null;
};
```
Think we can return `null` for better code consistency.